sql/item_basic.sql

```diff
@@ -5327,8 +5327,8 @@ INSERT INTO `item_basic` VALUES (5672,0,'dried_berry','dried_berry',12,1548,57,0
 INSERT INTO `item_basic` VALUES (5673,0,'rolsin','rolsin',12,1548,57,0,84);
 INSERT INTO `item_basic` VALUES (5674,0,'aileens_delight','aileens_delight',1,58968,0,0,3198);
 INSERT INTO `item_basic` VALUES (5675,0,'withered_rolanberry','withered_berry',1,58968,0,0,4624);
-INSERT INTO `item_basic` VALUES (5676,0,'serving_of_mushroom_sautee','mushroom_sautee',1,1580,54,0,764);
-INSERT INTO `item_basic` VALUES (5677,0,'serving_of_patriarch_sautee','patriarch_sautee',1,1580,54,0,955);
+INSERT INTO `item_basic` VALUES (5676,0,'serving_of_mushroom_saute','mushroom_saute',1,1580,54,0,4025);
```

Comment:
4025 on the NQ and 955 on the HQ?
#shitWikiSays

One or both of them is wrong, and I wouldn't put it past Joe random wiki editor to have put the price he sold it to an guid NPC for instead of a regular shop price. I'd rather keep a lower price unless I see retail confirmation on them both.

Reply:
Hmm, well, according to ffxiah:

> Mushroom: NPC ~ 4025 Gil
So I changed that one to match.
I checked ffxicyclopedia, for the HQ it says: Resale Price: 4,912~5,022 gil
For NQ, it says: Resale Price: 4,025~4,125 gil

I will adjust (since I didn't actually touch the HQ price since I did not see a price immediately, and didn't notice until you mentioned xD )

Reply:
I saw both of wiki's pages on it..And I don't trust what I see there in ffxiclopedia. The same SE that makes nearly everything sub1 is gonna put this random food between 4 and 5k? seems unlikely doesn't it? And sicne we normally add items with an unknown price at 0, that begs the question of where our current values came from - I haven't looked by in the history but maybe somebody checked this before.